openssl/src/crypto/pkcs5.rs

```diff
@@ -1,6 +1,42 @@
 use libc::c_int;
+use std::ptr::null;
+
+use crypto::symm_internal::evpc;
+use crypto::hash;
+use crypto::symm;
 use ffi;
 
+/// Derives a key and an IV from various parameters.
+/// New applications should use pbkdf2_hmac_sha1 or another more modern key
+/// derevation algorithm
+pub fn evp_bytestokey(typ: symm::Type, message_digest_type: hash::Type,
```

Comment:
Changed it to evp_bytes_to_key_pbkdf1_compatible. It's not great name but it's quite a weird function.

https://www.openssl.org/docs/crypto/EVP_BytesToKey.html